modern/tests/fixtures/expected.py

```diff
@@ -70,7 +70,7 @@ def get_compiler_versions():
 
 
 def get_envfile_values():
-    envfile = os.path.join(os.path.dirname(__file__), '..', '..', '.env')
+    envfile = os.path.join(os.path.dirname(__file__), '..', '..', '..', '.env')
```

Comment:
The folder `modern` should contain everything it needs. Remember that the purpose is to replace root. The file `modern/.env` should exist with all the required content.